config/deprecation-workflow.js

```diff
@@ -1,6 +1,6 @@
 window.deprecationWorkflow = window.deprecationWorkflow || {};
 window.deprecationWorkflow.config = {
     workflow: [
-        {handler: 'silence', matchMessage: 'You modified (-join-classes "ember-view" "form-group" (-normalize-class "errorClass" errorClass activeClass=undefined inactiveClass=undefined)) twice in a single render. This was unreliable in Ember 1.x and will be removed in Ember 3.0'}
+        {handler: 'silence', matchId: 'ember-runtime.action-handler-_actions'}
```

Comment:
Actually now that you mention it, I know what's causing these deprecations. In the file uploader we're using `invoke` instead of just `send`. That's what's causing the deprecations.